e2e/helm.e2e.spec.ts

```diff
@@ -0,0 +1,100 @@
+import path from 'path';
+import {
+  ElectronApplication, BrowserContext, _electron, Page, Locator
+} from 'playwright';
+import { test, expect } from '@playwright/test';
+import { createDefaultSettings, kubectl, helm } from './utils/TestUtils';
+
+let page: Page;
+const defaultReportFolder = path.join(__dirname, 'reports/');
+
+test.describe.serial('Helm Deployment Test', () => {
+  let mainTitle: Locator;
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+
+  const mainTitleSelector = '[data-test="mainTitle"]';
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        '--disable-dev-shm-usage',
+      ]
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+  });
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: path.join(defaultReportFolder, 'pw-trace.zip') });
+    await electronApp.close();
+  });
+
+  test('should load Rancher Desktop App', async() => {
```

Comment:
Doesn't this duplicate the test in `main.e2e.spec.ts`?

Reply:
Nope, we already have it in there, with a different name.